### PR TITLE
Toggle next / prev buttons on <amp-lightbox-gallery>

### PR DESCRIPTION
**Pull request by inidaname**
This PR fixes issues #13357 

@cathyxz

### CODE REVIEW

File: extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js [hunk 1]
```diff
@@ -296,6 +296,7 @@ export class AmpLightboxGallery extends AMP.BaseElement {
       this.carousel_.setAttribute('type', 'slides');
       this.carousel_.setAttribute('layout', 'fill');
       this.carousel_.setAttribute('loop', '');
+      this.carousel_.setAttribute('controls', 'controls');
```
Comment (cathyxz) [inline extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js]: You don't actually need to set it to `'controls'`, you can just set it to the empty string like in the line above. 

```
this.carousel_.setAttribute('controls', '');
```

File: extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js [hunk 2]
```diff
@@ -524,13 +525,15 @@ export class AmpLightboxGallery extends AMP.BaseElement {
     }
 
     if (this.controlsMode_ == LightboxControlsModes.CONTROLS_HIDDEN) {
+      this.carousel_.setAttribute('controls', 'controls');
```
Comment (cathyxz) [inline extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js]: Same here.

Comment (inidaname) [reply extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js]: Amended

**Comment by inidaname** [comment]
@cathyxz I sent in another commit with you suggested edit.

**Pull request merged**